src/parser/jobs/blm/RotationWatchdog.js

```diff
@@ -158,14 +166,15 @@ export default class FireCounter extends Module {
 
 	_resetRecording() {
 		this._inFireRotation = false
-		this._fireCounter = {}
+		this._rotation = {}
 		this._lockedBuffs = false
 	}
 
 	_getMissingFire4Count(count, hasConvert) {
 		const NotEnoughUH = this._UH  < 2
-		const missingFire4 = EXPECTED_FIRE4 + (hasConvert ? FIRE4_FROM_CONVERT : 0) - (NotEnoughUH ? 1 : 0) - count
-		return missingFire4
+		const ExpectedFire4 = EXPECTED_FIRE4 + (hasConvert ? FIRE4_FROM_CONVERT : 0) - (NotEnoughUH ? 1 : 0)
+		const missingFire4 = ExpectedFire4 - count
+		return {missing: missingFire4, expected: ExpectedFire4}
```

Comment:
might be cleaner to name the variables the same as the object keys, which would allow you do do something like

```javascript
const expected = [ ... ]
const missing = [ ... ]
return {missing, expected}
```